src/program/game/components/position/position.ts

```diff
@@ -8,7 +8,7 @@ export class Position extends ComponentAbstract<PositionInterface> {
         super(
             ComponentEnum.POSITION,
             {
-                [ComponentEnum.POSITION]: {
+                position: {
```

Comment:
won't the ComponentEnum already add the name space for the data ?

Reply:
But now is done on the "addComponent" inside the entity.abstract class